Fix build when BUILD_WEBENGINE=OFF

hefee opened the pull request:
Fix build when BUILD_WEBENGINE=OFF

hefee commented:
Please do not forget to cherry-pick to master.

claucambra reviewed (approved):
Thank you!

claucambra commented:
@hefee Could you sign-off the commit?

hefee commented:
> @hefee Could you sign-off the commit?

done.

nilsding reviewed (approved):
thanks for the contribution!

nilsding commented:
/backport to master

mgallien commented:
> Please do not forget to cherry-pick to master.

@hefee thanks
missed your comment
we do the opposite way so with a merge to master and a backport to stable branches
I changed the target branch. Can you rebase on top of it ?

hefee commented:
> > Please do not forget to cherry-pick to master.
> 
> @hefee thanks missed your comment we do the opposite way so with a merge to master and a backport to stable branches I changed the target branch. Can you rebase on top of it ?

rebased on master.

mgallien commented:
/backport to stable-3.16

hefee commented:
> Hello there, Thank you so much for taking the time and effort to create a pull request to our Nextcloud project.
> 
> We hope that the review process is going smooth and is helpful for you. We want to ensure your pull request is reviewed to your satisfaction. If you have a moment, our community management team would very much appreciate your feedback on your experience with this PR review process.
> 
> Your feedback is valuable to us as we continuously strive to improve our community developer experience. Please take a moment to complete our short survey by clicking on the following link: https://cloud.nextcloud.com/apps/forms/s/i9Ago4EQRZ7TWxjfmeEpPkf6
> 
> Thank you for contributing to Nextcloud and we hope to hear from you soon!
> 
> (If you believe you should not receive this message, you can add yourself to the [blocklist](https://github.com/nextcloud/.github/blob/master/non-community-usernames.txt).)

@mgallien: Please make the bot stop spamming me. One message is enough, but adding a comment every day is too much!
